Makefile

```diff
@@ -0,0 +1,16 @@
+release: clean
+	git checkout master
+	git pull origin master
+	python setup.py sdist bdist_wheel
+	python -m twine upload --sign --identity security@readthedocs.org dist/*
+
+tag:
+	git checkout master
+	git pull origin master
+	git tag `python -c "print(__import__('readthedocs_ext').__version__)"`
+	git push --tags
+
+clean:
+	rm -rf dist/
+
+.PHONY: release clean tag
```

Comment:
Seems like a good improvement. I would like to settle on more common tooling patterns across our repos going forward though. On the other repos we're using `inv` for this, which I quite like. That could maybe be a next step though. I'm generally a fan of `make` otherwise.

Reply:
+1 on standardization. I see that `release` depends on `clean` - not sure if invoke supports that sort of dependency tracking though. Brett Cannon asked on Twitter the other day and after a few recommendations, he went for doit https://twitter.com/brettsky/status/1382122878225686529

Reply:
Aren't we already using invoke for this?

Reply:
@astrojuanlu yeah, `doit` helps a lot with cached dependencies too. However, `invoke` supports pre-tasks http://docs.pyinvoke.org/en/latest/getting-started.html?highlight=pre#declaring-pre-tasks

I'm also 👍🏼 to avoid makefile and use `invoke` here.